tests/mesh_test_2d.cc

```diff
@@ -578,7 +578,7 @@ TEST_CASE("Mesh is checked for 2D case", "[mesh][2D]") {
         // Gauss point generation
         Json jgen;
         jgen["type"] = "gauss";
-        jgen["material_id"] = mid;
+        jgen["material_id"] = {0};
```

Comment:
We can no longer have a scalar value?

Reply:
Sure, can! The `mesh_test_3d.cc` test is still in scalar, I just add a variation for testing purposes.